ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_random.py

```diff
@@ -171,3 +171,62 @@ def test_gamma(
     for (u, v) in zip(ret, ret_gt):
         assert ivy.all(u >= 0)
         assert ivy.all(v >= 0)
+
+
+# poisson
+@handle_test(
+    fn_tree="functional.ivy.experimental.poisson",
+    dtype_and_lam=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float", full=False),
+        min_value=0,
+        max_value=5,
+        min_num_dims=0,
+    ),
+    shape=helpers.get_shape(
+        min_num_dims=1,
+        max_num_dims=3,
+        min_dim_size=1,
+        max_dim_size=10,
+    ),
+    dtype=helpers.get_dtypes("float", full=False),
+    seed=helpers.ints(min_value=0, max_value=100),
+)
```

Comment:
Please add a test_gradients=st.just(False) in the handle_test block as the function shouldn't support gradients. Even though the message only corresponds to the tensorflow function not supporting gradients, given that this is a random generation related function and it's not unified yet, we shouldn't test for gradients for any backend. Finally, just add a ToDo comment to enable gradient tests for this function with the other backends once random generation is unified